gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/ui/SnappyRegionStatsCollectorFunction.java

```diff
@@ -79,14 +79,20 @@ public void execute(FunctionContext context) {
       for (GemFireContainer container : containers) {
         if (container.isApplicationTable()) {
           LocalRegion r = container.getRegion();
+          String regionFullPath = r.getFullPath();
           if (managementService != null && r != null ) {
-            RegionMXBean bean = managementService.getLocalRegionMBean(r.getFullPath());
+            RegionMXBean bean = managementService.getLocalRegionMBean(regionFullPath);
             if (bean != null && !(r.getFullPath().startsWith(
                 "/" + Misc.SNAPPY_HIVE_METASTORE + '/'))) {
               SnappyRegionStats dataCollector = collectDataFromBean(r, bean);
               if (dataCollector.isColumnTable()) {
                 cachBatchStats.put(dataCollector.getTableName(), dataCollector);
               } else {
+                // Set as Column Table, if this table is corresponding row table of column store
```

Comment:
It will be better to consult the catalog from the lead node itself for column table or row table once the stats has come from everywhere. This is because on each node you don't need to find out for a particular table.

Reply:
As discussed, appropriate code changes are done on lead node side in Snappydata repository.
This PR is no more required and hence closing it.